pkgs/development/compilers/llvm/17/default.nix

```diff
@@ -150,6 +150,9 @@ in let
           src = ../common/clang/clang-at-least-16-LLVMgold-path.patch;
           libllvmLibdir = "${tools.libllvm.lib}/lib";
         })
+      ] ++ lib.optionals stdenv.targetPlatform.isOpenBSD [
+        # TODO make unconditional
```

Comment:
Is this a promise to submit a PR to staging making it unconditional as soon as this is merged to staging?

Reply:
If we switch to LLVM 18 perhaps it doesn't even need to go to staging!